diff --git a/yolov6/data/data_augment.py b/yolov6/data/data_augment.py
index 76c4b51..388ca79 100644
--- a/yolov6/data/data_augment.py
+++ b/yolov6/data/data_augment.py
@@ -78,7 +78,11 @@ def random_affine(img, labels=(), degrees=10, translate=.1, scale=.1, shear=10,
                   new_shape=(640, 640)):
     '''Applies Random affine transformation.'''
     n = len(labels)
-    height, width = new_shape
+    try:
+        height, width = new_shape
+    except:
+        height = new_shape
+        width = new_shape
 
     M, s = get_transform_matrix(img.shape[:2], (height, width), degrees, scale, shear, translate)
     if (M != np.eye(3)).any():  # image changed
